exampleCourse/questions/templateRandomMultipleChoiceListPrompt/question.html

```diff
@@ -0,0 +1,10 @@
+<pl-question-panel>
+{{{params.question_prompt}}}
```

Comment:
I actually think the previous way was easier to understand, and we should double-brace the substitution (not triple). If people want to put HTML in their substitutions then we should have a separate example for that.

Reply:
I aimed to use triple brackets to make the questions as robust as possible and to reduce any issues that might occur. (Who knows what will be feed in from the `server.py`?)